Cubical/Categories/Instances/FromEnriched.agda

```diff
@@ -0,0 +1,41 @@
+module Cubical.Categories.Instances.FromEnriched where
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+open import Cubical.Categories.Category
+open import Cubical.Categories.Monoidal.Base
+open import Cubical.Categories.Monoidal.Enriched
+open import Cubical.Categories.Monoidal.Instances.Presheaf
+open import Cubical.Categories.Enriched.Instances.Presheaf.Self
+open import Cubical.Categories.NaturalTransformation
+open Category
+open EnrichedCategory
+open NatTrans
+
+module _
+  {ℓ ℓ' ℓS : Level}
+  {C : Category ℓ ℓ'}
+  (E : EnrichedCategory (PshMon.𝓟Mon {ℓS = ℓS} C) ℓS) where
+
+  module E = EnrichedCategory E
+  module PMC = PshMon {ℓS = ℓS} C
+  module M = MonoidalCategory PMC.𝓟Mon
+
+
+  Enriched→Cat : Category _ _
```

Comment:
No change needed but maybe a useful observation: A category enriched in Sets is definitionally isomorphic to an ordinary Category, and then this construction is a combination of that with change of base.